src/cleartext.js

```diff
@@ -43,7 +43,7 @@ export function CleartextMessage(text, signature) {
     return new CleartextMessage(text, signature);
   }
   // normalize EOL to canonical form <CR><LF>
-  this.text = util.canonicalizeEOL(util.removeTrailingSpaces(text));
+  this.text = normalizeEOL(util.removeTrailingSpaces(text), true);
```

Comment:
Instead of doing two different things with one function, could you do something like
```suggestion
  this.text = normalizeEOL(util.removeTrailingSpaces(text)).replace(/[\r\n]/g, '\r\n');
```
or even inline `normalizeEOL` in both places (since it will then be very short)?